src/functions_math.jl

```diff
@@ -54,10 +54,14 @@ for A in (
     Adjoint{<:Number,<:AbstractVector},
     Transpose{<:Real,<:AbstractVector{<:Real}},
     Transpose{<:Any, <:AbstractMatrix{T}} where T,  # resolves ambiguity error in AxisKeys
+    Union{Adjoint{T, <:AbstractVector{T}}, Transpose{T, <:AbstractVector{T}}} where {T},
 )
     @eval function Base.:*(a::$A, b::NamedDimsArray{L,T,1,<:AbstractVector{T}}) where {L,T}
         return *(a, parent(b))
     end
+    @eval function Base.:*(b::NamedDimsArray{L,T,1,<:AbstractVector{T}}, a::$A) where {L,T}
```

Comment:
Do we need this one? I think the `CoVector` one above is meant to take care of these.